pkg/exec/test_http_client.go

```diff
@@ -0,0 +1,59 @@
+package exec
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+func NewTestHttpClient(timeout time.Duration, trips Trips) http.Client {
+	return http.Client{
+		Timeout:   timeout,
+		Transport: NewTestRoundTripper(trips...),
+	}
+}
+
+type Trips []trip
+
+type trip struct {
```

Comment:
I think I would export the type as it is part of the public interface.

Reply:
changed the naming a bit